CatProducer/python/jetProducer_cfi.py

```diff
@@ -2,12 +2,9 @@
 
 catJets = cms.EDProducer("CATJetProducer",
     src = cms.InputTag("selectedPatJetsPFlow"),
-    shiftedEnDownSrc = cms.InputTag("shiftedSlimmedJetsEnDown"),
-    shiftedEnUpSrc = cms.InputTag("shiftedSlimmedJetsEnUp"),
-    smearedResSrc = cms.InputTag("smearedSlimmedJets"),
-    smearedResDownSrc = cms.InputTag("smearedSlimmedJetsResDown"),
-    smearedResUpSrc = cms.InputTag("smearedSlimmedJetsResUp"),
-    btagNames = cms.vstring()
+    btagNames = cms.vstring(),
+    ## payloadName should be AK4PFchs, but PHYS14_25_V2 does not have uncertainty 
+    payloadName = cms.string(""), 
```

Comment:
it doesnt crash
in PHYS14_25_V2 they didnt add the uncertainties, while it exists in PHYS14_25_V1
Since no one is looking at uncertainties now anyways, I just left it as a place holder